### PR TITLE
first issue

### LINKED ISSUE #6: [US-0] Menu
## Value Statement
As a user I want to be able to open a navigation menu via a button and closed to navigation menu.

## Description
image

## Acceptance Criteria
- [x] The menu should be indicated by an appropriate icon.
- [x] The menu icon should be at the top left in page.
- [x] The button should open a navigation menu and closed navigation menu

## Tasks
- [x] Create feature branch
- [x] Build component
- [x] write test for button
- [x] Add Styles

## Complexity
low

### PR DESCRIPTION
**Pull request by suademre**
- Required libraries are downloaded.
- added Nav Component 
This resolves #6

### CODE REVIEW

File: src/components/Nav/Nav.spec.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import Nav from '.'
+
+describe('Nav', () => {
+  it('it renders', () => {
+    render(<Nav />)
+
+    const nav = screen.getByTestId('navButton')
```
Comment (macfrei) [inline src/components/Nav/Nav.spec.js:9]: Why do you need to get it via `testId` this should be our _last_ choice. Usually we get the button via `getByRole`.

File: src/components/Nav/Nav.stories.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import Nav from '.'
+
+export default { title: 'Nav', component: Nav }
+
+const Template = args => <Nav {...args} />
+
+export const menuOpen = Template.bind({})
+// More on args: https://storybook.js.org/docs/react/writing-stories/args
```
Comment (macfrei) [inline src/components/Nav/Nav.stories.js:8]: This comment should be deleted.

File: src/components/Nav/Nav.stories.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import Nav from '.'
```
Comment (macfrei) [inline src/components/Nav/Nav.stories.js:1]: I think the path is not correct?

File: src/components/Nav/index.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    const [showMenu, setShowMenu] = useState(false);
+
+    const toggleMenu = () => {
+        setShowMenu(!showMenu);
+    };
+
+    return (
+        <div>
+            <HeaderNav>
+                <div className="menu-icon" id="open">
```
Comment (macfrei) [inline src/components/Nav/index.js:15]: Do you still need the `className`? And the `id`?

File: src/components/Nav/index.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                    <button
+                        data-testid="navButton"
+                        aria-label="navButton"
```
Comment (macfrei) [inline src/components/Nav/index.js:18]: This is not a good `aria-label`. The aria-label should be descriptive and in real English/German etc. 
```suggestion
                        aria-label="Open navigation"
```

File: src/components/Nav/index.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    const toggleMenu = () => {
+        setShowMenu(!showMenu);
+    };
+
+    return (
+        <div>
+            <HeaderNav>
+                <div className="menu-icon" id="open">
+                    <button
+                        data-testid="navButton"
```
Comment (macfrei) [inline src/components/Nav/index.js:17]: You don't need this testId.

File: src/stories/Button.jsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```
Comment (macfrei) [inline src/stories/Button.jsx:2]: You can delete the `stories` folder.

File: src/assets/css/style.css [hunk 8]
```diff
@@ -0,0 +1,46 @@
+nav {
```
[the remark this answers is not in the document]
Comment (suademre) [reply src/assets/css/style.css:1]: no :) i deleted

File: src/components/Nav/Nav.spec.js (hunk 1 above)
Comment (suademre) [reply src/components/Nav/Nav.spec.js:9]: i changed with getAllByRole.

File: src/components/Nav/Nav.stories.js (hunk 2 above)
Comment (suademre) [reply src/components/Nav/Nav.stories.js:8]: i did.

File: src/components/Nav/index.js (hunk 5 above)
Comment (suademre) [reply src/components/Nav/index.js:18]: i did it

**Pull request merged**